android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactTVExoplayerView.java

```diff
@@ -375,6 +379,15 @@ private void createViews() {
         exoDorisPlayerView.setExoDorisPlayerViewListener(this);
         exoDorisPlayerView.setUseController(false);
 
+        // Watermark view.
+        if (exoDorisPlayerView.getChildCount() > 1) {
+            watermarkWidget = new DceWatermarkWidget(themedReactContext);
+            ViewGroup.LayoutParams params =
+                    new ViewGroup.LayoutParams(
+                            ViewGroup.LayoutParams.WRAP_CONTENT, ViewGroup.LayoutParams.WRAP_CONTENT);
+            exoDorisPlayerView.addView(watermarkWidget, 1, params);
```

Comment:
Why do we need to add it to index `1` instead of `0`?

Reply:
Also probably we can add it without any layout params, because that will be applied dynamically anyway in `DceWatermarkWidget#applyStyle(...)`, but need to be tested out first before removing it.